core/src/main/java/org/apache/calcite/rel/type/RelDataTypeSystemImpl.java

```diff
@@ -168,6 +171,8 @@ public abstract class RelDataTypeSystemImpl implements RelDataTypeSystem {
       return isPrefix ? "TIMESTAMP '" : "'";
     case TIMESTAMP_WITH_LOCAL_TIME_ZONE:
       return isPrefix ? "TIMESTAMP WITH LOCAL TIME ZONE '" : "'";
+    case INTERVAL_WEEK:
+      return isPrefix ? "INTERVAL '" : "' WEEK";
```

Comment:
I see that `INTERVAL_SECOND` is using day as well. Should week be using day here? Is `INTERVAL_SECOND` likely a typo?

Reply:
The only place I see `getLiteral` being used right now in Calcite is in `CalciteMetaImpl.java`, and I'm not sure it's relevant to our use of Calcite which might be why this typo is OK.